sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -191,7 +191,7 @@ internal class RadarLocationManager(
                 if (!foregroundService.updatesOnly) {
                     this.startForegroundService(foregroundService)
                 }
-            } else if (RadarForegroundService.started) {
+            } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && RadarForegroundService.started) {
```

Comment:
what is the reason why we do not want to call `this.stopForegroundService` if the sdk level is lower than `Build.VERSION_CODES.O`?